tests/schemas/valid/test_introduction_with_guidance.json

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
Not a blocker but a lot of content in this can be slimmed down. Most of the content isn't needed, placeholders etc.

Reply:
I will reduce it if time allows.